src/testers.jl

```diff
@@ -191,6 +192,9 @@ function test_rrule(
         ∂self = ∂s[1]
         x̄s_ad = ∂s[2:end]
         @test ∂self === NoTangent()  # No internal fields
+        msg = "The pullback must return `(∂self, ∂args...)`, where `length(∂args) == " *
+            "length(args)`, and `args` are the arguments of the primal function."
```

Comment:
Maybe more direct about the problem
```suggestion
            "The pullback should return 1 cotangent for each primal input."
```